rt/rt.mjs

```diff
@@ -203,14 +203,14 @@ class Module {
 }
 
 export class Engine {
-    constructor(filesystem) {
+    constructor(options) {
```

Comment:
Might as well use destructuring here:

```
constructor({filesystem}) { ... }
```

Can be fixed in a follow-up if you like.

Reply:
no problem! I just addressed it here.